Sources/ArcGISToolkit/Components/OfflineMapAreasView/OfflineMapAreasView.MapViewModel.swift

```diff
@@ -22,24 +22,17 @@ extension OfflineMapAreasView {
     @MainActor
     class MapViewModel: ObservableObject {
         /// The portal item ID of the web map.
-        private let portalItemID: String?
+        private let portalItemID: Item.ID?
```

Comment:
I think it's a good idea to continue to use `Item.ID` as the type for these properties instead of devolving to String